.devcontainer/Dockerfile

```diff
@@ -1,21 +1,23 @@
-FROM mcr.microsoft.com/devcontainers/python:1-3.11-bullseye
-
-RUN set -x
-
-RUN apt-get update \
-    && apt-get -y install curl \
-    && apt-get -y install net-tools \
-    && apt-get -y install procps \
-    && apt-get -y install build-essential \
-    && apt-get -y install docker.io
-
-RUN pip install ipython ipykernel
-RUN ipython kernel install --user --name dstoolkit
-
-COPY requirements.txt .
-
-RUN pip install -r requirements.txt
-
-RUN set +x
-
-CMD bash
+FROM python:3.12-bullseye
+
+RUN set -x
+
+RUN apt-get update \
+    && apt-get -y install curl \
+    && apt-get -y install net-tools \
+    && apt-get -y install procps \
+    && apt-get -y install build-essential \
+    && apt-get -y install docker.io

```

Comment:
Just wondering if we could pull these all in one command.. something like this?
```
RUN apt-get update \
    && apt-get -y install \
        curl \
        net-tools \
        procps \
        build-essential \
        docker.io
```

Reply:
Yeah I didn't change that but can done that way!

Reply:
updated